packages/react-core/src/components/Toolbar/__tests__/ToolbarToggleGroup.test.tsx

```diff
@@ -102,19 +102,19 @@ describe('Toolbar', () => {
     const customChipGroupContent = (
       <React.Fragment>
         <ToolbarItem>
-          <Button id="save-button" variant="link" onClick={() => {}} isInline>
+          <Button variant="link" onClick={() => {}} isInline>
             Save filters
           </Button>
         </ToolbarItem>
         <ToolbarItem>
-          <Button id="clear-button" variant="link" onClick={() => {}} isInline>
+          <Button data-testid="clear-button" variant="link" onClick={() => {}} isInline>
```

Comment:
Nit: I don't think this testId is needed any more.

```suggestion
          <Button variant="link" onClick={() => {}} isInline>
```